src/api/app/controllers/webui/project_controller.rb

```diff
@@ -48,7 +48,7 @@ def index
     @show_all = params[:show_all]
     projects = Project.all
     projects = projects.not_home unless @show_all
-    @projects = projects.pluck(:name, :title)
+    @projects = projects.select(:name, :title)
```

Comment:
``select`` is probably a little bit slower than ``pluck`` but for better readability I use select now. The index list is cached anyway, so I don't think this will cause performance issues.